Bottom: 8098571666685aa6a8d637532398e0f42a184f64
Top:    ee2ec1736de73c9f4e3690770e1545adb4e0c0f4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 20:14:27 +0100

Refresh of hid-gtk-better-encapsulate-som

---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index da96f57..b575816 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -327,6 +327,7 @@ ghid_layer_selector_init (GHidLayerSelector *ls)
   ls->accel_group = gtk_accel_group_new ();
   ls->action_group = gtk_action_group_new ("LayerSelector");
   ls->n_actions = 0;
+
   for (i = 0; i < 20; ++i)
     ls->accel_available[i] = TRUE;
